src/pages/workspaces/List.js

```diff
@@ -275,7 +278,20 @@ export const WorkspaceList = () => {
                   tooltip: !canView &&
                     'You cannot access this workspace because it is protected by an Authorization Domain. Click to learn about gaining access.',
                   tooltipSide: 'right'
-                }, [name])
+                }, [name]),
+                workspaceVersion === 'v1' && h(IdContainer, [
+                  id => h(TooltipTrigger, {
+                    side: 'right',
+                    type: 'light',
+                    useTooltipAsLabel: true,
+                    content: h(v1WorkspaceNotification, { id, showIcon: false, showLinks: false })
```

Comment:
We are not rendering links because tooltips are not focusable (can't click on the links with mouse or keyboard).